File: Projects/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void draw_pyramid(int);
int get_height();


int main(void)
{
    int height = get_height();
    draw_pyramid(height);
}


//function to ask user for height
int get_height()
{
    int height = 0;
    //ask the user for an input between 1-8
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    return height;
}

//function to draw the pyramid
void draw_pyramid(int num)
{
    for (int i = 1; i <= num; i++)
    {
        //add spaces before the first bricks
        for (int j = num - i; j > 0; j--)
        {
            printf(" ");
        }
        //add left side bricks
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        //add middle spaces
        printf("  ");
        //add right side bricks
        for (int l = 0; l < i; l++)
        {
            printf("#");
        }
        //move to next line
        printf("\n");
    }
}

File: Projects/sentimental-mario-more/mario.py
from cs50 import get_int

# Ask user for height only continuing when input is a number between 1 nad 8
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Print the pyramid
for x in range(1, height + 1):
    print((" " * (height - x)) + ("#" * x) + "  " + ("#" * x))